lib/evm-storage-verifier/src/lib.rs

```diff
@@ -76,7 +76,7 @@ pub fn verify_storage_absence(
 /// * `proof`: Proof of storage.
 /// * `storage_root`: Storage root of the contract.
 ///
-/// NOTE: You must not trust the `root` unless you verified it by calling [`validate_light_client_update`].
+/// NOTE: You must not trust the `root` unless you verified it by calling `ethereum_sync_protocol::validate_light_client_update`.
```

Comment:
```suggestion
/// NOTE: You must not trust the `root` unless it has been verified.
```

this link should just be removed, it's an artifact from when these two crates were one crate previously